mux/bindings/conformance/runner.py

```
@@ -0,0 +1,325 @@
+    commands = fixture.get("requires", {}).get("commands", [])
+    if not commands:
+        return
+    unsupported: List[str] = []
+    with MuxClient(socket_path=socket_path, timeout=3.0) as client:
+        for command in commands:
+            response = client.request(command)
+            error = str(response.get("error", ""))
+            if response.get("ok") is False and "unknown variant" in error:
+                unsupported.append(command)
+    if unsupported:
+        raise FixtureSkipped(f"server lacks required command(s): {', '.join(unsupported)}")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)